test/system/030-run.bats

```diff
@@ -736,4 +736,14 @@ EOF
     is "$output" "$random_1" "output matches STDIN"
 }
 
+# rhbz#1763007 : the --log-opt for podman run does not work as expected
+@test "podman run with log-opt option" {
+    run_podman run -d --rm --log-opt max-size=1m $IMAGE sleep 5
+    cid=$output
+    run_podman inspect --format "{{ .Config.CreateCommand }}" $cid
```

Comment:
Wouldn't it be better to check `{{.HostConfig.LogConfig.Size}}' instead? `CreateCommand` is just basically a parrot check; it doesn't actually confirm that the option was parsed properly.